azure_license-server/main.tf

```diff
@@ -0,0 +1,88 @@
+variable "azure_region" {}
+variable "resource_name" {}
+
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "rg" {
+  name     = var.resource_name
+  location = var.azure_region
+}
+
+resource "azurerm_virtual_network" "vnet" {
+  name                = "sentieon-vnet"
+  location            = azurerm_resource_group.rg.location
+  resource_group_name = azurerm_resource_group.rg.name
+  address_space       = ["172.31.0.0/16"]
+}
+
+resource "azurerm_subnet" "subnet" {
+  name                 = "sentieon-subnet"
+  virtual_network_name = azurerm_virtual_network.vnet.name
+  resource_group_name  = azurerm_resource_group.rg.name
+  address_prefixes     = ["172.31.0.0/24"]
+}
+
+module "security_group" {
```

Comment:
The file `security_group_rules.tf` needs to be in the `security_group` directory to call it as a module.